src/DotNet.Status.Web/BuildMonitorOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace DotNet.Status.Web
```

Comment:
They would be the only class in that namespace.  A namespace with only one class is specifically cautioned against.  I can prefix them, but the names get pretty crazy... AzurePipelinesMonitorOptionsBuildMonitorBuild...

I might move all the "Options" style classes into their own "Options" namespace (and folder), because they are sort of just splattered all over the root folder.  Maybe that'll look better.  Or maybe the "Options" should be nested classes of the things they are options for?

So `AzurePipelinesMonitorOptions` would become `AzurePipelinesController.Options.AzurePipelines`

Reply:
There, I tried some stuff, see if that feels better.